pkg/workload/tpcc/tpcc.go

```diff
@@ -137,6 +141,38 @@ func (w *waitSetter) String() string {
 	}
 }
 
+type FileLoggerQueryTracer struct {
+	filePath string
+}
+
+func (t FileLoggerQueryTracer) TraceQueryStart(
```

Comment:
maybe I'm missing something, but who calls these two functions here and how are ops actually written to the log file?

Reply:
From L664 in `pkg/go/mod/github.com/jackc/pgx/conn.go`:

```go
func (c *Conn) Query(ctx context.Context, sql string, args ...any) (Rows, error) {
	if c.queryTracer != nil {
		ctx = c.queryTracer.TraceQueryStart(ctx, c, TraceQueryStartData{SQL: sql, Args: args})
	}
```

Reply:
ohhhh, now i understand why you were asking about explicitly implementing interfaces yesterday lol. To make this a little clearer, could you add the following line above your struct?

```
var _ pgx.QueryTracer = FileLoggerQueryTracer{}
```

Reply:
Done